### DISCUSSION TITLE
✨ DocuSeal

### DISCUSSION THREAD

**Discussion by JigSawFr** [runtipi/runtipi-appstore#2186] [Requests]
### Description

DocuSeal is an open-source platform that provides secure and efficient digital document signing and processing. It allows users to create, fill, and sign PDF forms online on any device. DocuSeal can also be integrated into web and mobile apps to provide seamless document signing workflows.

- PDF form fields builder (WYSIWYG)
-  11 field types available (Signature, Date, File, Checkbox etc.)
-  Multiple submitters per document
-  Automated emails via SMTP
-  Files storage on disk or AWS S3, Google Storage, Azure Cloud
-  Automatic PDF eSignature
-  PDF signature verification
-  Users management
-  Mobile-optimized
-  API and Webhooks for integrations
-  Easy to deploy in minutes

### Official Website

https://www.docuseal.co/

### Documentation link

https://www.docuseal.co/docs

### Last application release & date

1.3.8 (15/01/2024)

### Application license

AGPL-3.0 license

### Source code repository link

https://github.com/docusealco/docuseal

### Docker image link

https://hub.docker.com/r/docuseal/docuseal

### Other's

**Demo:** https://demo.docuseal.co/

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

**Comment by JigSawFr** [comment]
Originally requested by @VitalyyBezuglyj

**Reply by JigSawFr** [reply]
Supported also by @user8446

**Comment by ivannovazzi** [comment]
DocuSeal is a solid self-hosted option for basic document signing. If anyone coming from this thread is specifically looking for a *hosted* solution that combines e-signature **+** client document collection in one flow (common in professional services — accountants, law firms, consultants who need clients to both sign an engagement letter *and* upload supporting files), [Gatherly](https://news.ycombinator.com/item?id=47085201) might be worth a look.

It sends clients a single Secure Link to a branded portal where they sign with eIDAS-qualified signatures and upload required documents — no account needed on the client side, automated reminders included. The main difference from DocuSeal is the integrated document request layer alongside the signing.
